test/files/prometheus-unit-tests/test.yml

```diff
@@ -142,3 +145,14 @@ tests:
         alertname: ElasticsearchProcessCPUHigh
         exp_alerts:
 
+      # --------- ElasticsearchCSVNotSuccessful ---------
+      - eval_time: 10m
+        alertname: ElasticsearchOperatorCSVNotSuccessful
```

Comment:
Would "ElasticSearchNotReconciled" be more descriptive from a user perspective?

Reply:
I think its descriptive enough, we also would be pointing to documentation from the message of the alert (in the future)

Reply:
for what it's worth, i don't have any attachment to `ElasticsearchOperatorCSVNotSuccessful` as an alertname. if we want to change it, i'm happy to do so